### DISCUSSION TITLE
Microsoft Support of Semantic Link Labs

### DISCUSSION THREAD

**Discussion by bgdszepesi** [microsoft/semantic-link-labs#303] [General]
Sorry - I have to ask this question explicitly since I know I will be asked by customers, will Microsoft Support take cases where this code is in use or is it considered 'Not Supported'?

**Comment by m-kovalsky** [comment]
At the moment, the official support for semantic link labs is me. Contacting Microsoft Support will likely end up in a few circles back to this GitHub page. Semantic Link is officially supported by Microsoft (and Microsoft Support).
